fusedrug/utils/cluster/cluster_using_mmseqs.py

```diff
@@ -0,0 +1,156 @@
+from fuse.data.ops import get_function_call_str
+import shutil
+import hashlib
+import subprocess
+from fuse.utils.file_io import save_text_file_safe, read_text_file
+import os
+
+def cluster(force_rebuild:bool = False, **kwargs):
```

Comment:
add "-> None" to specify there are not return values in the functions in this file.

Reply:
thanks - I'll modify it to return a dict with the important generated filenames